lib/glimesh_web/templates/oauth2_provider/application/index.html.eex

```diff
@@ -0,0 +1,28 @@
+<div class="container" style="text-align:center;">
+  <h1><%= gettext("Your applications") %></h1>
+
+  <table style="width:100%">
+    <thead>
+      <tr class="row">
+        <th class="col-md-2"><%= gettext("Name") %></th>
+        <th class="col-md-8"><%= gettext("Callback URL") %></th>
+        <th class="col-md-2"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for application <- @applications do %>
+        <tr class="row">
+          <td class="col-md-2 list-group-item"><%= link application.name, style: "width: 100%; padding: 0.4375rem 0rem; text-shadow: none; font-size: 14px; color: #007bff; font-weight: 400; white-space: normal; word-wrap: break-word; touch-action: manipulation; cursor: pointer; display: inline-block; text-allign: center; vertical-allign: middle; user-select: none; background-color:#0c1321; border-radius: 0.25rem;", to: Routes.application_path(@conn, :show, application) %></td>
```

Comment:
Can you move this to our CSS file, or figure out what bootstrap classes you can use to not have any custom styling?